src/PoolManager.sol

```diff
@@ -347,6 +347,17 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, Claims {
         emit ProtocolFeeUpdated(id, newProtocolFees);
     }
 
+    function collectProtocolFees(address recipient, Currency currency, uint256 amount)
```

Comment:
This function is basically a function that allows users to collect the underlying collateral for their claim - yet its locked down to something only 1 address is allowed to do.
I think it me it would make more sense to make the `protocolFeeController` have to do whatever all other users have to do to collect their claims.
So either:

- get a lock
- call `burn`
- call `take`

Or open this function up to be a function anyone can call, on the Claim contract

Reply:
Yep so initially I was planning on having protocol fee controller have to lock to collect fees like a normal user of Claims but I opted for this approach to keep the most backward compatibility with the current implementation.

I don't think we should open up an additional function in core to burn Claims for ERC20s since we already require lock/burn/take. I'm down to enforce that feeController locks to collect fees

Reply:
I like exposing this outside of lock to keep backwards compatibility